RampUpAuto/src/main/java/seleautomation/com/RampUpAuto/App.java

```diff
@@ -1,34 +1,36 @@
 package seleautomation.com.RampUpAuto;
 
+import org.openqa.selenium.By;
+import org.openqa.selenium.Keys;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.chrome.ChromeDriver;
+import io.github.bonigarcia.wdm.WebDriverManager;
+import org.openqa.selenium.support.ui.WebDriverWait;
+
+import java.time.Duration;
+import java.util.concurrent.TimeUnit;
 /**
  * Hello world!
  *
  */
 public class App 
-{
-    public static void main( String[] args )
-    {
-
-
-        public int add (int x, int y){
-
-        return x+y;
-    }
-
-        public int subtract (int x, int y){
-
-        return x-y;
-    }
-
-        public int multiply (int x, int y){
-
-        return x*y;
-    }
-        public int divide (int x, int y){
-
-        return x /y;
-    }
-
+{WebDriver driver;
+        WebDriverManager.chromedriver().setup();
+    // driver.chromedriver().setup();
+    driver = new ChromeDriver();
+
+        driver.get("https://www.google.com");
+        driver.manage().window().maximize();
+        driver.manage().timeouts().implicitlyWait(30, TimeUnit.SECONDS);

```

Comment:
Try to remove it please , no need of it, as we always trying to limit the time delays in the code as possible as we can